tellor_all_languages/test/test_file_contents_changed.js

```diff
@@ -0,0 +1,194 @@
+var urllib = require('urllib');
+
+const CompareFileContents = artifacts.require("./CompareFileContents.sol");
+const Tellor = artifacts.require("TellorPlayground.sol");
+
+//Helper function that submits and value and returns a timestamp for easy retrieval
+const submitTellorValue = async (tellorOracle, requestId, amount) => {
+  //Get the amount of values for that timestamp
+  let count = await tellorOracle.getNewValueCountbyRequestId();
+  await tellorOracle.submitValue(requestId, amount);
+  let time = await getTimestampbyRequestIDandIndex(requestId, count.toString());
+  return time.toNumber();
+};
+
+contract("UsingTellor Tests", function (accounts) {
+	let compareFileContents;
+	let tellorOracle;
+
+	beforeEach("Setup contract for each test", async function () {
+	tellorOracle = await Tellor.new();
+	compareFileContents = await CompareFileContents.new(tellorOracle.address);
+	});
+
+	it("Update Price", async function () {
+	const requestId = 1; // assumes the first ID of the list of tellor variable contains what we return, e.g. usd/BTC (in our case, checkflag text)
+	// TODO: change the requestId to two strings: 
```

Comment:
Todo's should not be merged into master.

Reply:
The TODO's are not a priority any more as they focused on NodeJS details of the code miner simulation code whereas Tellor miner code is written in GoLang.